examples/config/ci/coordinator.json

```diff
@@ -0,0 +1,102 @@
+        "Legend": "Percent",
+        "Query": "(sum(rate(mattermost_api_time_count{status_code=~\"5..\"}[1m]))/sum(rate(mattermost_api_time_count[1m])))*100",
+        "Threshold": 0.025,
+        "MinIntervalSec": 60,
+        "Alert": true
+      },
+      {
+        "Description": "Average client request duration",
+        "Legend": "Avg duration (s)",
+        "Query": "sum(rate(loadtest_http_request_time_sum[1m]))/sum(rate(loadtest_http_request_time_count[1m]))",
+        "Threshold": 0.1,
+        "MinIntervalSec": 60,
+        "Alert": true
+      },
+      {
+        "Description": "99th percentile of client request duration",
+        "Legend": "P99 duration (s)",
+        "Query": "histogram_quantile(0.99, sum(rate(loadtest_http_request_time_bucket[1m])) by (le))",
+        "Threshold": 2.0,
+        "MinIntervalSec": 60,
+        "Alert": true
+      },
+      {
+        "Description": "Percentage of HTTP 5xx client errors",
+        "Legend": "Percent",
+        "Query": "(sum(rate(loadtest_http_errors_total{status_code=~\"5..\"}[1m]))/sum(rate(loadtest_http_request_time_count[1m])))*100",
+        "Threshold": 0.025,
+        "MinIntervalSec": 60,
+        "Alert": true
+      },
+      {
+        "Description": "Percentage of client timeouts",
+        "Legend": "Percent",
+        "Query": "(sum(rate(loadtest_http_timeouts_total[1m]))/sum(rate(loadtest_http_request_time_count[1m]))) * 100",
+        "Threshold": 0.025,
+        "MinIntervalSec": 60,
+        "Alert": true
+      },
+      {
+        "Description": "CPU utilization - Average of app nodes",
+        "Legend": "Percent",
+        "Query": "100 - 100 * (avg(irate(node_cpu_seconds_total{instance=~\"app.*\",mode=\"idle\"}[5m])))",
+        "Threshold": 85,
+        "MinIntervalSec": 60,
+        "Alert": true
+      },
+      {
+        "Description": "Memory utilization - Average of app nodes",
+        "Legend": "Percent",
+        "Query": "100 - 100 * avg(node_memory_MemAvailable_bytes{instance=~\"app.*\"} / node_memory_MemTotal_bytes{instance=~\"app.*\"})",
+        "Threshold": 85,
+        "MinIntervalSec": 60,
+        "Alert": true
+      },
+      {
+        "Description": "Percentage of TCP retransmissions in the app nodes",
+        "Legend": "Percent",
+        "Query": "(avg(rate(node_netstat_Tcp_RetransSegs{instance=~\"app.*\"}[1m])) / avg(rate(node_netstat_Tcp_OutSegs{instance=~\"app.*\"}[1m]))) * 100",
+        "Threshold": 0.5,
+        "MinIntervalSec": 60,
+        "Alert": true
+      },
+      {
+        "Description": "Percentage of TCP retransmissions in the proxy node",
+        "Legend": "Percent",
+        "Query": "(avg(rate(node_netstat_Tcp_RetransSegs{instance=~\"proxy:9100\"}[1m])) / avg(rate(node_netstat_Tcp_OutSegs{instance=~\"proxy:9100\"}[1m]))) * 100",
+        "Threshold": 0.5,
+        "MinIntervalSec": 60,
+        "Alert": true
+      }
+    ]
+  },
+  "NumUsersInc": 8,
+  "NumUsersDec": 8,
+  "RestTimeSec": 1,
```

Comment:
Just noting this doesn't make too much sense if `UpdateIntervalMs` is still at 2s.

Reply:
Just beware that if we change the logic underneath (e.g. https://mattermost.atlassian.net/browse/MM-61922), then we could get different results from one version to the next.

Reply:
Yup, that's important. There's still room for increasing this, though, I tested it with 12 users/s (which would actually be 6 users/s because `UpdateIntervalMs` was still at 2s) many many times. I'll keep an eye out, though.